src/ott/solvers/linear/sinkhorn.py

```diff
@@ -42,6 +44,21 @@
 
 __all__ = ["Sinkhorn", "SinkhornOutput", "solve"]
 
+ProgressCallbackFn = Callable[
```

Comment:
We usually postfix types with `_t` to distinguish from class names, can call this `ProgressCallbackFn_t`?